cluster-provision/okd/scripts/provision.sh

```diff
@@ -13,6 +13,17 @@ if [ ! -z $INSTALLER_RELEASE_IMAGE ]; then
     done
 fi
 
+MASTERS=1
+if [[ $INSTALLER_TAG == "release-4.4" ]]; then
+    MASTERS=3
+fi
+
+function oc_retry {
+    until oc $@; do
```

Comment:
Can we parametrize this with a timeout?

Reply:
no need imo atm, because for the ones that need timeout i use timeout time bash -c
and there i cant use functions because i use bash
those are meant for infinite retries on parts that are expected to finish after several retries